crates/javy/src/lib.rs

```diff
@@ -40,15 +40,115 @@
 //!   and MessagePack byte slices
 
 pub use config::Config;
-pub use quickjs_wasm_rs as quickjs;
+pub use rquickjs as quickjs;
 pub use runtime::Runtime;
 
 pub mod alloc;
 mod config;
 mod runtime;
 
+use anyhow::{anyhow, Error, Result};
+use rquickjs::{prelude::Rest, Ctx, Error as JSError, Exception, Type, Value};
+use std::fmt::Write;
+
 #[cfg(feature = "messagepack")]
 pub mod messagepack;
 
 #[cfg(feature = "json")]
 pub mod json;
+
+/// Print the given JS value.
+///
+/// The implementation matches the default JavaScript display format for each value.
+pub fn print<'js>(val: &Value<'js>, sink: &mut String) -> Result<()> {
+    match val.type_of() {
+        Type::Undefined => write!(sink, "undefined").map_err(Into::into),
+        Type::Null => write!(sink, "null").map_err(Into::into),
+        Type::Bool => {
+            let b = val.as_bool().unwrap();
+            write!(sink, "{}", b).map_err(Into::into)
+        }
+        Type::Int => {
+            let i = val.as_int().unwrap();
+            write!(sink, "{}", i).map_err(Into::into)
+        }
+        Type::Float => {
+            let f = val.as_float().unwrap();
+            write!(sink, "{}", f).map_err(Into::into)
+        }
+        Type::String => {
+            let s = val.as_string().unwrap();
+            write!(sink, "{}", s.to_string()?).map_err(Into::into)
+        }
+        Type::Array => {
```

Comment:
I guess it could, but I think that part of a bigger conversation though, given that there are many conventions that we could follow, for now I've kept it similar to the previous implementation:

```js
console.log([1, 2, 4]);
```


```
❯ npx javy-cli compile index.js -o index.wasm
❯ wasmtime run --invoke=_start index.wasm
1,2,4
```

Reply:
Not saying that we shouldn't do it, mostly that it's probably outside of the scope of this change.